nio/src/test/scala/zio/nio/channels/ChannelSpec.scala

```diff
@@ -13,7 +13,7 @@ object ChannelSpec
       suite("ChannelSpec")(
         testM("read/write") {
           val inetAddress: ZIO[Any, Exception, InetSocketAddress] = InetAddress.localHost
-            .flatMap(iAddr => SocketAddress.inetSocketAddress(iAddr, 13370))
+            .flatMap(iAddr => SocketAddress.inetSocketAddress(iAddr, 13380))
```

Comment:
try to pass 0 as port that should pick first available

Reply:
For some reason this fails. The tests error out completely with a `BindException`.